src/components/business-card/business-card.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import styles from './styles.module.scss';
+import PrimaryButton from '../primary-button/primary-button';
+import useLikeCard from '../../hooks/use-like-card';
+import clsx from 'clsx';
+
+const BusinessCard = ({ id, src, alt, serviceCategory, serviceName, personName, address }) => {
+  const { addLikedCard, removeLikedCard, isCardLiked } = useLikeCard();
+
+  const handleLike = () => {
```

Comment:
Šis patikrinimas turi būti viduje hook'o useLikeCard, ir išeksportuota tik funkcija toggleLikeCard